libfreerdp/codec/rfx_neon.c

```diff
@@ -225,6 +225,296 @@ static void rfx_dwt_2d_decode_NEON(INT16* buffer, INT16* dwt_buffer)
 	rfx_dwt_2d_decode_block_NEON(buffer, dwt_buffer, 32);
 }
 
+static INLINE void __attribute__((__gnu_inline__, __always_inline__, __artificial__))
```

Comment:
remove `__attribute__`, we try not to assume any compiler (well, not 100% possible but this is already inline, so the keywords are mostly for debugging, right?)